NSrtm.Core/Pgm/Grid/IPgmGrid.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NSrtm.Core.Pgm.Grid
+{
+    public interface IPgmGrid
+    {
+        double GetClosestUndulationValue(double latitude, double longitude);
```

Comment:
For who is this method designed? We should rather not expose such approximation directly.